Makefile

```diff
@@ -135,7 +135,7 @@ endif
 endif
 
 .PHONY: test-checkstyle
-test-checkstyle: test-checkstyle-standalone test-tidy-compile
+test-checkstyle: test-checkstyle-standalone test-tidy-compile test-containers-compose
```

Comment:
As I learned from cdywan the compose test is a dynamic, functional test, not a static style check so please include that not as part of "checkstyle" but as part of the dynamic tests, e.g. above in line 131+133